client/src/screens/BallotScreen.tsx

```diff
@@ -37,7 +38,9 @@ const BallotScreen = () => {
           })}
         </Monospace>
       </p>
-      <p>Print page to view ballot.</p>
+      <p>
+	<Button onPress={window.print}>Print Ballot</Button>
```

Comment:
Not sure if this is preferred?
```suggestion
	<Button onPress={(window.kiosk ?? window).print}>Print Ballot</Button>
```

Reply:
not needed, kiosk intercepts automatically.

Reply:
> not needed, kiosk intercepts automatically.

I don’t think that’s true. See https://github.com/votingworks/kiosk-browser/blob/master/src/preload.ts

Reply:
Your code doesn't, but the default Electron code does and provides a print dialog, which is great for this purpose. We could choose to later do more, but for now this works pretty well.